src/views/about/page.tsx

```diff
@@ -1,18 +1,171 @@
-import Header from "@/components/layouts/PageHeader/PageHeader";
-
-const About = () => {
-  return (
-    <>
-      <Header
-        pageHeading="College Alumni"
-        subHeading="Welcome College alumni and undergraduates! We hope you will explore the many ways to connect with the NIT Arunachal Pradesh community."
-        bgImage="/header-bg/2023-06-07-1.jpg"
-      />
-      <div className="__page-content container">
-        <h1>About</h1>
-      </div>
-    </>
-  );
-};
-
-export default About;
+import Header from "@/components/layouts/PageHeader/PageHeader";
+import styles from "./About.module.scss";
+
+const About = () => {
+  return (
+    <>
+      <Header
+        pageHeading="College Alumni"
+        subHeading="Welcome College alumni and undergraduates! We hope you will explore the many ways to connect with the NIT Arunachal Pradesh community."
+        bgImage="/header-bg/2023-06-07-1.jpg"
+      />
+      <div className="__page-content container">
+
+      <section className={styles.__pageContentContainer}>
+        <h1>About</h1>
+
+        <div className={styles.body}>
```

Comment:
Make the page responsive for smaller screens

Reply:
I've made some adjustments and commit it, but it seems the cards got resized in the end. I'll get this sorted ASAP. Meanwhile, you can keep track of the progress, and pleases let me know if there's any further changes required.